packages/workflow/.eslintrc.js

```diff
@@ -11,8 +11,19 @@ module.exports = {
 	rules: {
 		complexity: 'error',
 
+		'import/order': [
```

Comment:
can we define these rules in the base config instead?

Reply:
Yes. That requires either fixing all `@n8n/` packages or turning the rule off for them. I'll move it the base config and fix those packages in a follow-up PR, if that's OK?

Reply:
Here: https://github.com/n8n-io/n8n/pull/10852